fabric-loot-api-v2/src/main/java/net/fabricmc/fabric/api/loot/v2/LootTableEvents.java

```diff
@@ -116,4 +125,14 @@ public interface Modify {
 		 */
 		void modifyLootTable(ResourceManager resourceManager, LootManager lootManager, Identifier id, LootTable.Builder tableBuilder, LootTableSource source);
 	}
+
+	public interface Loaded {
+		/**
+		 * Called when all loot tables have been loaded and {@link LootTableEvents#REPLACE} and {@link LootTableEvents#MODIFY} have been invoked.
+		 *
+		 * @param resourceManager the server resource manager
+		 * @param lootManager     the loot manager
+		 */
+		void onLootTableLoaded(ResourceManager resourceManager, LootManager lootManager);
```

Comment:
```suggestion
		void onLootTablesLoaded(ResourceManager resourceManager, LootManager lootManager);
```
technically there's multiple of them? :P

Reply:
Maybe then the event should be called `ALL_LOADED`? Previous events are called per loot table, so the new event may be confusing.

Also I think this event should be after the `MODIFY`/`REPLACE` events in the code, but that does not matter, it is about checkstyle.

In fact, this event is very similar to [CommonLifecycleEvents.TAGS_LOADED](https://github.com/FabricMC/fabric/blob/670b2ea08de60d952f62d5333f93bfe18c27f8ed/fabric-lifecycle-events-v1/src/main/java/net/fabricmc/fabric/api/event/lifecycle/v1/CommonLifecycleEvents.java#L31)

Reply:
> Also I think this event should be after the MODIFY/REPLACE events in the code, but that does not matter, it is about checkstyle.

wdym by that?

Reply:
> wdym by that?

Ah, sorry, nevermind. I am inattentive